frontend/src/pages/releases/fall23.mdx

```diff
@@ -0,0 +1,79 @@
+import clsx from 'clsx';
+import styles from './fall23.module.css';
+import { TextComponent } from '../../components/StyledComponents';
+
+
+<div className={clsx(styles.container, 'mx-auto')}>
+<TextComponent type={1}>
+
+# CourseTable 23Fall/Winter Release Notes
+
+_2024-01-21_
+
+Hi everyone!
+
+Thank you so much for all of your support and feedback this year. We had an amazingly productive semester working on changes for you all. We have had over **300** commits to our codebase since the end of school year – in past years this number has only been around 50! We have been used by almost all members of the Yale undergraduate community, and in the past month have processed over **5 million** requests.
+
+![Graph of CourseTable requests over time. The Total Requests is 5.26M.](/src/images/release_notes/fall_23/fall23requests.png)
```

Comment:
@bearsyankees You need to put it in `public` and link to it without the `public` part.